goimports is not part of the tools.go

As a result the `make ci` fails:

See: https://github.com/drpaneas/romie/pull/63/checks?check_run_id=1226695043

```shell
Run make ci
Downloading tools and dependencies...
main.go
curl -sfL https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh | sh
goreleaser/goreleaser info checking GitHub for latest tag
goreleaser/goreleaser info found version: 0.143.0 for v0.143.0/Linux/x86_64
goreleaser/goreleaser info installed ./bin/goreleaser
Building...
Running tests...
?   	romie	[no test files]
Checking the imports...
/bin/sh: 1: goimports: not found
Makefile:35: recipe for target 'fmt' failed
make: *** [fmt] Error 127
Error: Process completed with exit code 2.
```

You need to add this to `tools.go` and into the `main.go` via `go generate`.

--- a/.gitattributes
+++ b/.gitattributes
@@ -1,0 +1,1 @@
+*.go text eol=lf 

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -15,46 +15,46 @@ jobs:
       matrix:
         os: [ubuntu-20.04, windows-latest, macos-latest]
     steps:
-      -
-        name: Checkout out source code
+      - name: Checkout out source code
         uses: actions/checkout@v2.3.3
         with:
           fetch-depth: 0
-      -
-        name: Set up Go environment
+          
+      - name: Set up Go environment
         uses: actions/setup-go@v2.1.3
         with:
           go-version: 1.15
-      -
-        name: Cache Go modules
+          
+      - name: Cache Go modules
         uses: actions/cache@v2.1.1
         with:
           path: ~/go/pkg/mod
           key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
           restore-keys: |
             ${{ runner.os }}-go-
-      -
-        name: make setup
+            
+      - name: make setup
         run: |
           make setup
-      -
-        name: make lint
+          
+      - name: make lint
         run: |
           make lint
-      -
-        name: make build
+          
+      - name: make build
         run: |
           make build
-      -
-        name: make test
+          
+      - name: make test
         run: |
           make test
+          
       - name: Upload binary
         uses: actions/upload-artifact@v2
         with:
           name: romie ${{ matrix.os }}
           path: ${{ github.workspace }}
-      -
-        name: make run
+          
+      - name: make run
         run: |
           make run

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -1,0 +1,60 @@
+name: build
+
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - 'v*'
+  pull_request:
+
+jobs:
+  Building:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, windows-latest, macos-latest]
+    steps:
+      -
+        name: Checkout out source code
+        uses: actions/checkout@v2.3.3
+        with:
+          fetch-depth: 0
+      -
+        name: Set up Go environment
+        uses: actions/setup-go@v2.1.3
+        with:
+          go-version: 1.15
+      -
+        name: Cache Go modules
+        uses: actions/cache@v2.1.1
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      -
+        name: make setup
+        run: |
+          make setup
+      -
+        name: make lint
+        run: |
+          make lint
+      -
+        name: make build
+        run: |
+          make build
+      -
+        name: make test
+        run: |
+          make test
+      - name: Upload binary
+        uses: actions/upload-artifact@v2
+        with:
+          name: romie ${{ matrix.os }}
+          path: ${{ github.workspace }}
+      -
+        name: make run
+        run: |
+          make run

--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
 .vscode/
 coverage.txt
 romie
+bin/goreleaser

--- a/Makefile
+++ b/Makefile
@@ -9,27 +9,27 @@ export GO111MODULE := on
 export GOPROXY = https://proxy.golang.org,direct
 
 # go source files
-SRC = $(shell find . -type f -name '*.go')
+SRC = $(shell find . -type f -name "*.go")
 # The name of the executable (default is current directory name)
 TARGET := $(shell echo $${PWD-`pwd`})
 
-.PHONY: all build setup fmt test cover lint ci clean todo run help
+.PHONY: all build setup fmt test cover lint clean todo run help
 
 ## all: Default target, now is build
 all: build
 
-## build: Builds the binary 
-build:
+## build: Builds the binary
+build: fmt
 	@echo "Building..."
 	@$(GOCMD) build -o ${NAME}
 
 ## setup: Runs mod download and generate
 setup:
 	@echo "Downloading tools and dependencies..."
-	@$(GOCMD) mod download
+	@$(GOCMD) mod download -x
 	@$(GOCMD) generate -v ./...
 
-## fmt: Runs go goimports and go fmt
+## fmt: Runs go goimports and gofmt
 fmt: setup
 	@echo "Checking the imports..."
 	@$(GOCMD)imports -w ${SRC}
@@ -45,17 +45,14 @@ test:
 cover: test
 	@$(GOCMD) tool cover -html=coverage.txt
 
-## lint: Runs golangci-lint (configuration at .golangci.yml) and misspell 
+## lint: Runs golangci-lint (configuration at .golangci.yml) and misspell
 lint: setup
 	@echo "Running linters..."
 	@golangci-lint run --max-issues-per-linter 0 --max-same-issues 0 ./...
 	@misspell ./...
 
-## ci: Runs steup build, test, fmt, lint
-ci: setup build test fmt lint 
-
 ## clean: Runs go clean
-clean: 
+clean:
 	@echo "Cleaning..."
 	@$(GOCMD) clean
 

--- a/main.go
+++ b/main.go
@@ -15,6 +15,7 @@ limitations under the License.
 */
 //go:generate go install github.com/golangci/golangci-lint/cmd/golangci-lint
 //go:generate go install github.com/client9/misspell/cmd/misspell
+//go:generate go install golang.org/x/tools/cmd/goimports
 
 package main
 

--- a/tools.go
+++ b/tools.go
@@ -7,4 +7,6 @@ import (
 	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
 	// Import misspell
 	_ "github.com/client9/misspell/cmd/misspell"
+	// import goimports
+	_ "golang.org/x/tools/cmd/goimports"
 )

